msteams-platform/m365-apps/specify-runtime-requirements.md

```diff
@@ -0,0 +1,131 @@
+description: Specify your app's runtime requirements in the app manifest to provide a deliberate experience on supported Microsoft 365 hosts. 
+ms.date: 7/10/2024
+ms.author: mosdevdocs
+author: erikadoyle
+ms.topic: conceptual
+ms.localizationpriority: medium
+ms.subservice: m365apps
+---
+# Specify Microsoft 365 host runtime requirements in your app manifest (preview)
+
+> [!NOTE]
+>
+> The ability to specify Microsoft 365 host runtime requirements in app manifest is currently in [public developer preview](../resources/schema/manifest-schema-dev-preview.md) and supported in Teams and Outlook mobile hosts.
+
+If you have a Teams app that contains a personal tab or message extension, you can easily [extend it to run across additional Microsoft 365 hosts](./overview.md), including Outlook and Microsoft 365 (Office) app across web, desktop, and mobile experiences. When you upgrade your Teams app to use app manifest version 1.13 or higher, your app is available in other Microsoft 365 application hosts by default. However, if your app also includes components that aren't yet supported across other Microsoft 365 hosts, your app might only partially load, resulting in unplanned and unsupported user experiences.
+
+To ensure intentional and deliberate customer experiences with your app, you can specify your app's runtime requirements in the app manifest to tailor its behavior in applicable Microsoft 365 hosts, or omit it from surfacing in contexts that you're not ready to support. Describing your app's runtime requirements in the app manifest also helps ensure your production-ready app experience will reach wider audiences in additional Microsoft 365 host applications in the future.
+
+Specifying your app's runtime requirements is useful in scenarios such as:
+
+- **One-way dependencies**: If your app includes both a tab and message extension, and the tab functions as a settings page for users to configure message extension functionality, you can ensure that your settings tab doesn't load in hosts that don't support your message extension.
+
+- **Mutual dependencies:** When your app has specific app components that must be loaded together in order to function properly, you can ensure your app is *only available* in Microsoft 365 hosts that support all components.  
+
+- **Capability requirements:** When your app has component parts with runtime requirements that aren't supported across all applicable Microsoft 365 hosts, you can ensure those components are *seamlessly omitted* (rather than loaded, yet not functional) from your app experience running in those hosts.
+
+## Microsoft 365 host support
+
+Only a subset of Microsoft 365 host applications supports the ability to specify runtime requirements in the app manifest. This support will expand over time. The following hosts ensure only applicable apps and their applicable components are made available to users:
+
+|Microsoft 365 host application| Web | Desktop | Mobile |
```

Comment:
We should probably add information about all the hosts and status (not available / coming soon).

Reply:
@juna-gupta, @ali-ms I added M365 app and Copilot to the table. Any other hosts I should add as rows?

Reply:
No. It should be T/O/H + Copilot.